### ISSUE TITLE
Cannot convert to TFRecords

### ISSUE THREAD

**Issue by pathakrohit08** [tensorflow/tensorflow#21581]
I have three text files `A.txt, B.txt,C.txt` the labels are `A,B and C`

    
    def convert(file_paths, labels, out_path):

        print("Converting: " + out_path)

        # Number of images. Used when printing the progress.
        num_files = len(file_paths)

        # Open a TFRecordWriter for the output-file.
       with tf.python_io.TFRecordWriter(out_path) as writer:
    
        # Iterate over all the image-paths and class-labels.
        for i, (path, label) in enumerate(zip(file_paths, labels)):
               # Print the percentage-progress.
               print_progress(count=i, total=num_files - 1)

               lines=getModifiedLines(path)

              # Create a dict with the data we want to save in the
              # TFRecords file. You can add more relevant data here.
              data = \
                  {
                      'text': wrap_int64(lines),
                      'label': wrap_int64(label)
                  }

              # Wrap the data as TensorFlow Features.
              feature = tf.train.Features(feature=data)

              # Wrap again as a TensorFlow Example.
              example = tf.train.Example(features=feature)

              # Serialize the data.
              serialized = example.SerializeToString()
              writer.write(serialized)

    def getModifiedLines(filePath):
        data = open(filePath, 'r', encoding='UTF8', errors='ignore').read()
        lines = re.split("\n", data)
        all_lines=[]
        for line in lines:
          _l=count_vect.fit_transform(line)
          all_lines.append(_l)

    return all_lines

 I am getting error at line `'text': wrap_int64(lines),`

>Error
`[<12071x21108 sparse matrix of type '<class 'numpy.int64'>'
	with 226655 stored elements in Compressed Sparse Row format>] has type 
   <class 'list'>, but expected one of: (<class 'int'>,)`

Have I written custom code  **Yes**
OS Platform and Distribution  **Ubuntu 16.04**
TensorFlow installed from   **https://storage.googleapis.com/tensorflow/linux/gpu/tensorflow_gpu-1.10.0-cp27-none-linux_x86_64.whl**

TensorFlow version  **1.9**
Bazel version 
CUDA/cuDNN version   **i think 7**
GPU model and memory **NVIDIA Titan V 12288MB**
Mobile device  **No**

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 17 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 32 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by pathakrohit08** [comment]
Since I never got an response I found an alternative . I am ok with this issue being closed.Thanks
